Main_WT_Server_DAQ.py

```diff
@@ -24,6 +68,12 @@
 
 # Read serial data from the CANdapter
 async def rec_serial_data(serial_connection):
+    current_time = time.time()
+    last_time = current_time
+    # buffer_time = 1/500
```

Comment:
What's this?

Reply:
This is used to keep track of the buffer time to ensure that we send data at a specified rate. Used to calculate delta time between loops.